src/_guides/language/concurrency/index.md

```diff
@@ -312,107 +311,144 @@ to implement isolates.
 }
 ```
 
-The relevant statement is the last one, which exits the isolate,
-sending `jsonData` to the passed-in `SendPort`.
-Message passing using `SendPort.send` normally involves data copying,
-and thus can be slow.
-However, when you send the data using `Isolate.exit()`,
-then the memory that holds the message in the exiting isolate isn’t copied,
-but instead is transferred to the receiving isolate.
-The sender will nonetheless perform a verification pass to ensure
-the objects are allowed to be transferred.
-
-
-{{site.alert.version-note}}
-  `Isolate.exit()` was added in 2.15.
-  Previous releases support only explicit message passing,
-  using `Isolate.send()` as shown in the next section's example.
-{{site.alert.end}}
-
-The following figure illustrates the communication between
-the main isolate and the worker isolate:
+This example accomplishes the same as the previous:
+a new isolate spawns and executes a computation, sending back the result.
 
-![A figure showing the previous snippets of code running in the main isolate and in the worker isolate](/guides/language/concurrency/images/isolate-api.png)
+However, now the isolate sends a [closure][]. 
+This allows versatility not only for how the isolate can be utilized
+(as closures are less limited than typical named functions),
+but also for the written style of the code.
+Here, `Isolate.run()` executes what looks like local code, concurrently.
+In that sense, you can imagine `run()` almost like a [control flow operator][]
+for “run in parallel”.
 
+[closure]: /guides/language/language-tour#anonymous-functions
+[control flow operator]: /guides/language/language-tour#control-flow-statements
 
 ### Sending multiple messages between isolates
 
-If you need more communication between isolates,
-then you need to use the [`send()` method][] of `SendPort`.
+`Isolate.run()` abstracts a handful of lower-level, 
+isolate-related API to simplify isolate management:
+
+* [`Isolate.spawn()`][] and [`Isolate.exit()`][]
+* [`ReceivePort`][] and [`SendPort`][]
+
+[`Isolate.exit()`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/Isolate/exit.html
+[`Isolate.spawn()`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/Isolate/spawn.html
+[`ReceivePort`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/ReceivePort-class.html
+[`SendPort`]: {{site.dart-api}}/{{site.data.pkg-vers.SDK.channel}}/dart-isolate/SendPort-class.html
+
+However, there are cases where it’s necessary
+to directly utilize the primitives that make up `run()`
+for more granular control over isolate management.
+Sending multiple messages between isolates is one of those cases.
+
+For more communication between isolates,
+you need to use the [`send()` method][] of `SendPort`.
 One common pattern, which the following figure shows,
 is for the main isolate to send a request message to the worker isolate,
 which then sends one or more reply messages.
```

Comment:
**question:** What does "for _more_ communication" mean? I would recommend finding a way to proffer an example as part of the explanation.

Reply:
The section header is "Sending multiple messages between isolates", that's what "more communication" is in reference to. The context is that one message is the standard, more than one requires different configuration. I'll figure out how to include that